src/frontend/src/flows/addDevice/manage/pollForTentativeDevice.ts

```diff
@@ -87,7 +120,8 @@ export const pollForTentativeDevice = (
         countdown.stop();
         resolve("canceled");
       },
-      link: addDeviceLink({ userNumber }),
+      origin: window.origin,
```

Comment:
not sure what does this change?

Reply:
Ah, instead of taking the whole link, the component now takes the `origin` and computes the link itself. It simplified the code a bit.

Reply:
Coolio, thanks for the explanation